### PR TITLE
docs: Tiny fixes for auto-scaling executors docs

### PR DESCRIPTION
**Pull request by mrnugget**
Stuff I ran into while trying to set this up on my machine.

### CODE REVIEW

File: doc/admin/deploy_executors.md [hunk 1]
```diff
@@ -278,40 +278,61 @@ Auto scaling makes use of the auto-scaling capabilities of the respective cloud
...
+For auto scaling to work, the Sourcegraph instance (its `worker` service, specifically) needs to be configured with the correct credentials in order to publish a scaling metric to the used cloud provider. The `credentials` submodule in both our [AWS](https://sourcegraph.com/github.com/sourcegraph/terraform-aws-executors/-/tree/modules/credentials) and [GCP](https://sourcegraph.com/github.com/sourcegraph/terraform-google-executors/-/tree/modules/credentials) executor modules exists for that purpose. When used, the `credentials` moduel sets up the credentials on the cloud provider and returns them in the terraform oujtputs it, you get properly configured credentials in the Terraform outputs.
```
Comment (eseliger) [inline doc/admin/deploy_executors.md:281]:
```suggestion
For auto scaling to work, the Sourcegraph instance (its `worker` service, specifically) needs to be configured with the correct credentials in order to publish a scaling metric to the used cloud provider. The `credentials` submodule in both our [AWS](https://sourcegraph.com/github.com/sourcegraph/terraform-aws-executors/-/tree/modules/credentials) and [GCP](https://sourcegraph.com/github.com/sourcegraph/terraform-google-executors/-/tree/modules/credentials) executor modules exists for that purpose. When used, the `credentials` module sets up the credentials on the cloud provider and returns them in the terraform outputs it, you get properly configured credentials in the Terraform outputs.
```

File: doc/admin/deploy_executors.md [hunk 2]
```diff
@@ -278,40 +278,61 @@ Auto scaling makes use of the auto-scaling capabilities of the respective cloud
...
+
+Here's an example of how one would use the `credentials` submodule:
 
 ```terraform
-module "credentials" {
+module "my-credentials" {
   source  = "sourcegraph/executors/<cloud>//modules/credentials"
+
+  # Find the latest version here:
```
Comment (eseliger) [inline doc/admin/deploy_executors.md:289]: Do we have this in other places where we say <version> too?

File: doc/admin/deploy_executors.md [hunk 3]
```diff
@@ -278,40 +278,61 @@ Auto scaling makes use of the auto-scaling capabilities of the respective cloud
...
+  # - https://github.com/sourcegraph/terraform-google-executors/tags
+  # - https://github.com/sourcegraph/terraform-aws-executors/tags
   version = "<version>"
 
   region          = <region>
   resource_prefix = ""
 }
+
+# For Google:
+output "metric_writer_credentials_file" {
```
Comment (eseliger) [inline doc/admin/deploy_executors.md:299]: Good catch!!

**Review by eseliger — APPROVED** [review]
Thank you!!

File: doc/admin/deploy_executors.md (hunk 2 above)
Comment (mrnugget) [reply doc/admin/deploy_executors.md:296]: No, I checked. Other places reference the Terraform files directly. But I really like having the snippet here.

**Comment by mrnugget** [comment]
I added another commit that explicitly mentions `min_replicas` and `max_replicas` which are required for this to work.

**Pull request merged**